src/components/Task/Drawer/CommentList/CommentList.tsx

```diff
@@ -53,7 +53,9 @@ const TaskDrawerCommentList = ({
   return (
     <>
       <Box m={2}>
-        {loading ? (
+        {error ? (
+          <p>{error.toString()}</p>
```

Comment:
This is very bare bones, not sure how we want to render error messages

Reply:
Christian was using the snackbar. https://github.com/CruGlobal/mpdx-react/pull/40/files#diff-c2a85f1c4f401282f7b2bffdfc7e006980106f80113c4e0668b8d798ea616643R260

Seems ok. We haven't discussed any best practices but I assume Tatai was the one who brought that in. Not sure how that would work during render... Maybe it needs a useEffect (or a custom hook to watch it and call the snackbar). Or we could make an error icon component with a message that could be rendered in the body of the list.